libs/ui/src/lib/breadcrumbs/breadcrumbs.component.stories.ts

```diff
@@ -7,15 +7,20 @@ export default {
   component: BreadcrumbsComponent,
   parameters: { notes },
   args: {
-    breadcrumbLabels: ['Avaleht', 'Abiellumine', 'Avalduse esitamine'],
+    breadcrumbs: [
+      { title: 'Avaleht', href: '/avaleht' },
+      { title: 'Ilma urlita samm' },
```

Comment:
When the branch is run locally, no label appears in the Storybook story for mobile: 
<img width="402" alt="image" src="https://github.com/e-gov/cvi/assets/1833029/d7b88455-57a5-464f-9d85-db4f4f0ce67f">

Is it expected, and if yes, has it been discussed with a designer? Maybe instead of showing nothing if the last breadcrumb item doesn't a `href`, it could be better to have a user navigate to the last item that has it. Eg. if the navigation structure is "Avaleht (has url) -> Some other page (has no url) -> Third page (has no url)", then when on "Third page" a user will be shown "< Avaleht" on mobile.

Reply:
Hmm, it does show "Abiellumine" for me. And your description is exactly what I was trying to do (except show the one before last, as last is the current page). That's why I added the mobile-breadcrumb class as well - the logic wasn't either possible or would have been hard to understand on css side.